flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/metrics/FlinkDeploymentMetrics.java

```diff
@@ -94,6 +101,22 @@ public void onUpdate(FlinkDeployment flinkApp) {
                         })
                 .add(deploymentName);
 
+        // Minor version computed from the above
+        var subVersions = flinkVersion.split("\\.");
+        String minorVersion = MALFORMED_MINOR_VERSION;
```

Comment:
Not sure why not use `UNKNOWN` just like the other case but I'm fine with that.

Reply:
Fair, I am open to simply using `UNKNOWN`.